demisto_sdk/commands/common/hook_validations/field_base_validator.py

```diff
@@ -496,7 +496,9 @@ def is_aliased_fields_are_valid(self) -> bool:
         """
 
         if not self.id_set_file:
-            print_warning("Validate will skip since an id set file was not provided")
+            logger.info(
```

Comment:
Not warning?

Reply:
Aligned: https://github.com/demisto/demisto-sdk/pull/2707/commits/d5dda30d8720bc13c0e8d6e808c20cf96eced4d9